doc/cascadia/profiles.schema.json

```diff
@@ -812,6 +812,16 @@
               "type": "string"
             }
           ]
+        },
+        "windowingBehavior": {
+          "default": "useNew",
+          "description": "Controls how new terminal instances attach to existing windows.",
+          "enum": [
+            "useNew",
+            "useAny",
+            "useAnyExisting"
```

Comment:
these two string values aren't as clear as their descriptions; should we make them a little more self-evident?

Reply:
Whoops, yea I changed those before I opened Teams to  to check what we had settled on 😋